linode_api4/groups/monitor.py

```diff
@@ -145,3 +148,218 @@ def create_token(
                 "Unexpected response when creating token!", json=result
             )
         return MonitorServiceToken(token=result["token"])
+
+    def get_alert_definitions(
```

Comment:
Can you change the function name to reflect it's for listing only? To align with the naming style in python sdk, the word `list` should be omitted.
```suggestion
    def alert_definitions(
```